docs/add-ins/outlook-on-send-addins.md

```diff
@@ -13,7 +13,7 @@ The on send feature for Outlook add-ins provides a way to handle email or block
 - Add a specific recipient to the CC line.
 
 > [!NOTE]
-> The on send feature is currently supported for Outlook on the web in Office 365 (in [preview](/office/dev/add-ins/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview)), Exchange 2016 on-premises (Cumulative Update 6 or later), and Exchange 2019 on-premises (Cumulative Update 1 or later). This feature is also available in [preview](/office/dev/add-ins/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview) on Windows and Mac connected to an Office 365 subscription. Add-ins that use the on send feature aren't allowed in [AppSource](https://appsource.microsoft.com).
+> The on send feature is currently supported for Outlook on the web in Office 365, Exchange 2016 on-premises (Cumulative Update 6 or later), and Exchange 2019 on-premises (Cumulative Update 1 or later). This feature is also available on Windows and Mac connected to an Office 365 subscription.
```

Comment:
I think "on send" should have a hyphen ("on-send").

Reply:
I think it's that too. Will scrub.